experimental/ModStd/ModStdQt.jl

```diff
@@ -517,11 +517,14 @@ The return value is an array
       and the multiplicity
 
 # Examples     
-```jldoctest
-julia>  Qa, a = PolynomialRing(QQ, :a=>1:2);
-julia>  R, X = PolynomialRing(FractionField(Qa), :X=>1:2);
-julia>  f = (X[1]^2+a[1]*X[2]^2)*(X[1]+X[2]);
-julia>  z = factor_absolute(f)
+```julia
+julia> Qa, a = PolynomialRing(QQ, :a=>1:2);
+
+julia> R, X = PolynomialRing(FractionField(Qa), :X=>1:2);
+
+julia> f = (X[1]^2+a[1]*X[2]^2)*(X[1]+X[2]);
+
+julia> z = factor_absolute(f)
```

Comment:
I do not thing this PR fixes the abs factor: some factors can be returned in the wrong parent. Maybe the example should be changed to this and the code fixed?
```
julia> Qa, a = PolynomialRing(QQ, :a=>1:2);

julia> R, X = PolynomialRing(FractionField(Qa), :X=>1:2);

julia> f = (X[1]^2+a[1]*X[2]^2)*(X[1]+X[2]+a[1]+a[2]);

julia> z = factor_absolute(f)
3-element Vector{Any}:
 1
 (x1 + x2 + x3 + x4, 1)
 (X1 + t*X2, X1 - t*X2, 1)

julia> parent(z[2][1])
Multivariate Polynomial Ring in x1, x2, x3, x4 over Rational Field
```

Reply:
Fixed the example.